dozer/cogs/actionlogs.py

```diff
@@ -485,6 +503,85 @@ async def help(self,
         """
         await ctx.send(embed=e)
 
+    @command()
+    @has_permissions(manage_nicknames=True)
+    @bot_has_permissions(manage_nicknames=True)
+    async def locknickname(self, ctx, member: discord.Member, name: str):
```

Comment:
Good idea might be to make `name` optional which will just default to the current member's nickname if not specified.